src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -8,7 +8,6 @@ slug: /projects/hardware/overview
 
 The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
 
-![Coronal image of the NUgus robot](../images/front-robot.png)
-![Sagittal image of the NUgus robot](../images/side-robot.png)
+![Coronal and saggital images of the NUgus robot](../images/NUgus.png)
```

Comment:
Maybe remove this change and re-add when you do the hardware pages.

Reply:
Done :+1: